versions.sh

```diff
@@ -75,15 +75,14 @@ for version in "${versions[@]}"; do
 					empty # trailing comma hack
 				| ., "slim-" + .), # https://github.com/docker-library/ruby/pull/142#issuecomment-320012893
 				(
-					# Alpine 3.17 defaults to OpenSSL 3 which is not supported (yet?) by Ruby 2.7 or 3.0
+					# Alpine 3.17+ defaults to OpenSSL 3 which is not supported (yet?) by Ruby 2.7 or 3.0
```

Comment:
The optimistic `yet?` has now resolved and OpenSSL 3 will not be supported in Ruby 2.7 or 3.0 (which also means that they will not work in `debian:bookworm` images):

> We decided to not backport openssl-3.x gem into `ruby_2_7` and `ruby_3_0` branches. Because it affects with OpenSSL 1.0.x.
>
>Unfortunately, the users should upgrade their ruby version to 3.1+ before upgrading their OS version.
>
> \- https://bugs.ruby-lang.org/issues/18658#note-18

Side note: 2.7 is already EOL (PR incoming) and 3.0 will be EOL in ~10 months (https://www.ruby-lang.org/en/downloads/branches/, https://endoflife.date/ruby)